packages/admin-panel/src/importExport/ImportModal.js

```diff
@@ -101,6 +101,7 @@ export const ImportModalComponent = React.memo(
         changeSuccess();
       } catch (error) {
         setStatus(STATUS.ERROR);
+        setFinishedMessage(null);
```

Comment:
@EMcQ-BES snuck in an extra one liner for good measure (deals with https://linear.app/bes/issue/NOT-798/survey-response-import-validation-not-working-correctly-success)

Re-review plz 🙏

Reply:
After I said how much I love a one liner, you add additional lines.